### DISCUSSION TITLE
Ansible And Vagrant Automation (anyone have idea about this error?)

### DISCUSSION THREAD

**Discussion by thelllmike** [community/community#68749] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

user@web-2022432:~/vagrant$ vagrant up --provision
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'ubuntu/trusty64' version '20190514.0.0' is up to date...
==> default: Running provisioner: ansible...
    default: Running ansible-playbook...
PYTHONUNBUFFERED=1 ANSIBLE_FORCE_COLOR=true ANSIBLE_HOST_KEY_CHECKING=false ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o IdentityFile=/mnt/c/Users/Micheal/lab/vagrant/.vagrant/machines/default/virtualbox/private_key -o ControlMaster=auto -o ControlPersist=60s' ansible-playbook --connection=ssh --timeout=30 --extra-vars=ansible_user\=\'vagrant\' --limit="default" --inventory-file=inventory.ini -v ubuntu.yml
Using /etc/ansible/ansible.cfg as config file

PLAY [default] *****************************************************************

TASK [Gathering Facts] *********************************************************
fatal: [192.168.56.10]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.56.10 port 2222: Connection refused", "unreachable": true}

PLAY RECAP *********************************************************************
192.168.56.10              : ok=0    changed=0    unreachable=1    failed=0    skipped=0    rescued=0    ignored=0

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.

**Comment by vishalkrishna8** [comment]
Hi  @thelllmike , If you're using Windows, ensure that you're running the command prompt or terminal as an administrator, as it may require elevated privileges.

**Reply by thelllmike** [reply]
im use wsl2 ubuntu 20.04

**Reply by vishalkrishna8** [reply]
You can try the following:
1.Ensure that the virtual machine is running and properly provisioned.
2.Check if the SSH service is running inside the virtual machine.
3.Verify that the SSH port configured in your Ansible playbook matches the port used by the virtual machine.
4.Review your firewall and network settings to ensure they are not blocking the connection.
